.github/workflows/automated-ci.yml

```diff
@@ -15,17 +15,60 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
```

Comment:
What is the purpose of this step?

Reply:
it was useful for fetching more data on getting diffs on push event, forgot to remove.